src/Sulu/Bundle/AdminBundle/Metadata/SchemaMetadata/ConstMetadata.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\AdminBundle\Metadata\SchemaMetadata;
+
+class ConstMetadata extends PropertyMetadata
+{
+    /**
+     * @var string|number|null
+     */
+    private $value;
```

Comment:
does it make sense that this is nullable?

Reply:
At first I thought you were right, but when thinking about it again I would leave it like this, but also return a schema if `null` is a returned. This way we could also say that a value has to be `null` (who knows if we need that one day?)

Reply:
Although what I've just described is not working because if I remove the null check in the `toJsonSchema` method of this class, then all the assigned properties say that `null` is their constant value, and the validation fails for a lot :see_no_evil: I would leave it like it is at the moment.

Reply:
yeah returning a schema in the case of `null` it makes sense 🙂